go/samples/http/Dockerfile

```diff
@@ -1,25 +1,14 @@
-# Build Stage
 # First pull Golang image
-FROM golang:1.19-alpine as build-env
+FROM golang:1.19-alpine
 
-ENV APP_NAME sqlcommenter-http
-ENV CMD_PATH main.go
- 
-# Copy application data into image
-COPY . $GOPATH/src/$APP_NAME
+# Set workdir
 WORKDIR $GOPATH/src/$APP_NAME
```

Comment:
Is there a default variable for APP_NAME as it is deleted by this commit in L5 ?

Reply:
Also did you run this application using `docker compose build && docker compose up` ?

Reply:
fixed